modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -1800,6 +1800,15 @@ public T getCell(int index) {
         return accumCell;
     }
 
+    /**
+     * The amount of pixels by which to adjust the position when the the scroll bar track is clicked.
+     *
+     * @return the value in pixels
+     */
+    public double getBlockIncrement() {
```

Comment:
I'd rather not add that method and make `getViewportLength()` public. That would benefit other use cases, and it is more behavior-agnostic. The "viewportLength" is easier to define in a contract than a "blockIncrement" and since this is exactly what you return, it sounds like a reasonable thing to do.

Reply:
Good suggestion. I have made the necessary changes.